spec/computable/registry/claim-reward.spec.ts

```diff
@@ -0,0 +1,129 @@
+  deployDll,
+  deployAttributeStore,
+  maybeParseInt,
+  eventReturnValues,
+} from '../../../src/helpers'
+
+const provider:any = ganache.provider(),
+  web3 = new Web3(provider)
+
+let erc20:Erc20,
+  dll:Contract,
+  store:Contract,
+  voting:Voting,
+  parameterizer:Parameterizer,
+  registry:Registry,
+  owner:string,
+  applicant:string,
+  challenger:string,
+  voter:string,
+  proposer:string
+
+describe('Registry: Claim Reward', () => {
+  beforeEach(async () => {
+    [owner, applicant, challenger, voter, proposer] = await web3.eth.getAccounts()
+
+    erc20 = new Erc20(owner)
+    const tokenAddress = await erc20.deploy(web3)
+    erc20.setProvider(provider)
+
+    dll = await deployDll(web3, owner)
+    dll.setProvider(provider)
+    const dllAddress = dll.options.address
+
+    store = await deployAttributeStore(web3, owner)
+    store.setProvider(provider)
+    const attributeStoreAddress = store.options.address
+
+    voting = new Voting(owner)
+    const votingAddress = await voting.deploy(web3, { tokenAddress, dllAddress, attributeStoreAddress })
+    voting.setProvider(provider)
+
+    parameterizer = new Parameterizer(owner)
+    const parameterizerAddress = await parameterizer.deploy(web3, { tokenAddress, votingAddress })
+    parameterizer.setProvider(provider)
+
+    registry = new Registry(owner)
+    const registryAddress = await registry.deploy(web3, { tokenAddress, votingAddress, parameterizerAddress, name: NAME })
+    registry.setProvider(provider)
+
+    //owner approves voting and reg to spend
+    // await erc20.approve(votingAddress, 1000000)
+    await erc20.approve(registryAddress, 1000000)
+
+    // applicant needs funding
+    await erc20.transfer(applicant, 500000)
+    await erc20.approve(registryAddress, 250000, { from: applicant })
+    await erc20.approve(parameterizerAddress, 250000, { from: applicant })
+
+    // challenger needs funding
+    await erc20.transfer(challenger, 500000)
+    await erc20.approve(registryAddress, 250000, { from: challenger })
+    await erc20.approve(parameterizerAddress, 250000, { from: challenger })
+
+    // voter needs funding
+    await erc20.transfer(voter, 600000)
+    await erc20.approve(registryAddress, 200000, { from: voter })
+    await erc20.approve(parameterizerAddress, 200000, { from: voter })
+    await erc20.approve(votingAddress, 200000, { from: voter })
+
+    // funds for proposer
+    await erc20.transfer(proposer, 100000)
+    await erc20.approve(parameterizerAddress, 100000, { from: proposer })
+  })
+
+  it('should transfer the correct number of tokens once a challenge has been resolved', async () => {
+    const listBytes = stringToBytes(web3, 'listing.net'),
+      voterStartingBalance = maybeParseInt(await erc20.balanceOf(voter))
+
+    //// Apply
+    const tx1 = registry.apply(listBytes, ParameterDefaults.MIN_DEPOSIT, '', { from: applicant })
+    expect(tx1).toBeTruthy()
+
+    //// Challenge
+    const challID = eventReturnValues('_Challenge',
```

Comment:
eventReturnValues is from the older http style specs, there are websocket listeners now rike `onData`. you can find them in the other specs

Reply:
This is stymieing me a bit. Can we punt this fix to a different PR?